atomic_reactor/plugins/pre_add_help.py

```diff
@@ -84,4 +91,4 @@ def run(self):
 
         self.log.info("added %s", man_path)
 
-        return content
+        return self.HELP_GENERATED
```

Comment:
> return None if exception and an empty string if no help file was used

Why do we need to distinguish between these two cases?

> duplicating information from the config

It's not really duplicating information in the config. This plugin decides which filename to use; I think it makes perfect sense for it to return the value of the decision.